bedrock/pocket/templates/pocket/add.html

```diff
@@ -26,84 +26,88 @@
-               <li><a href="https://getpocket.com/apps/windowsphone/" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">Windows Mobile</a></li>
-               <li><a href="https://getpocket.com/apps/blackberry/" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">Blackberry</a></li>
-               <li><a href="https://getpocket.com/apps/" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">More Apps</a></li>
+               <li><a href="https://chrome.google.com/webstore/detail/save-to-pocket/niloccemoadcdkdjlinkgdfekeahmflj?hl=en" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">{{ ftl('pocket-add-windows') }}</a></li>
+               <li><a href="https://help.getpocket.com/article/1006-getting-started-with-kobo" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">{{ ftl('pocket-add-kobo-e-reader') }}</a></li>
+               <li><a href="https://getpocket.com/apps/windowsphone/" class="dark-inline-link rarrow">{{ ftl('pocket-add-windows-mobile') }}</a></li>
+               <li><a href="https://getpocket.com/apps/blackberry/" class="dark-inline-link rarrow">{{ ftl('pocket-add-blackberry') }}</a></li>
+               <li><a href="https://getpocket.com/apps/" class="dark-inline-link rarrow">{{ ftl('pocket-add-more-apps') }}</a></li>
             </ul>
          </div>
       </div>
    </section>
 
    <section class="add-save">
       <div class="add-save-wrapper">
-            <h2>Unlimited Ways to Save</h2>
+            <h2>{{ ftl('pocket-add-unlimited-ways') }}</h2>
          <div class="save-context">
             <div class="save-email">
-               <h3>Save via e-mail</h3>
-               <p>email any link to your list by sending it to <a href="mailto:add@getpocket.com" class="dark-inline-link">add@getpocket.com</a> </p>
-               <div class="save-email-image">
-                  <img src="{{ static('img/pocket/save-via-email.png') }}" alt="">
-               </div>
-               <a href="https://help.getpocket.com/article/1020-saving-to-pocket-via-email" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">Learn more</a>
+                <h3>{{ ftl('pocket-add-save-via-email') }}</h3>
+                <p>{{ ftl('pocket-add-email-any-link', add_email='add@getpocket.com',add_email_link='mailto:add@getpocket.com', add_email_css_class='dark-inline-link') }}</p>
+                <a class="save-email-link" href="http://help.getpocket.com/customer/portal/articles/482759">
+                  <div class="save-email-card">
+                    <p class="save-email-card-line email"><span>{{ ftl('pocket-add-to') }}: </span>add@getpocket.com</p>
+                    <p class="save-email-card-line"><span>{{ ftl('pocket-add-subject') }}: </span>{{ ftl('pocket-add-inca-trail') }}</p>
+                    <p class="save-email-card-line no-border">http://www.YourLinkHere.com</p>
```

Comment:
Issue (blocking): Nice work on making the strings in this image localisable! I think it's a good idea to keep this URL out of Fluent, as we should probably keep control the URL that we display here.

I do think we should change this however, as visiting this redirects to something naughty and NSFW 🙈

Using www.example.com should be safe as it can't be purchased: 

```suggestion
                    <p class="save-email-card-line no-border">https://www.example.com/</p>
```

Reply:
oh noooo! Thanks for checking 🙈